src/frontend/Sider/Behandling/VedtakOgBeregning/Læremidler/validering.ts

```diff
@@ -1,8 +1,10 @@
+import { Vedtaksperiode } from './InnvilgeVedtak/InnvilgeLæremidler';
 import { FormErrors } from '../../../../hooks/felles/useFormState';
 import { ÅrsakAvslag, ÅrsakOpphør } from '../../../../typer/vedtak/vedtak';
-import { Periode, PeriodeMedEndretKey, validerPeriode } from '../../../../utils/periode';
+import { Periode, validerPeriode } from '../../../../utils/periode';
 import { harIkkeVerdi } from '../../../../utils/utils';
 
+// TODO: Dette er pliss likt barnetilsyn - fikse?
```

Comment:
Burde dette være en oppgave i stedet for en todo i koden? 👼

Reply:
Fjerna den akkurat! 🚀